kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/executor/CommandStatus.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eurotech and/or its affiliates
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.executor;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.apache.commons.io.output.NullOutputStream;
+
+/**
+ * 
+ * The CommandStatus object is returned by the {@link CommandExecutorService} after the execution of a command.
+ * It contains all the relevant informations about the result of the command execution.
+ * <p>
+ * The parameters are the following:
+ * <ul>
+ * <li>exitStatus : the {@link ExitValue} of the command. A value other than 0 means an error. When the command is
+ * stopped by timeout the exit value is 124.</li>
+ * <li>isTimedout : a flag that signals that the command was stopped by timeout</li>
+ * <li>outputStream : the output of the command</li>
+ * <li>errorStream : the error stream of the command</li>
+ * <li>inputStream : the input stream used to send data to the process</li>
+ * </ul>
+ *
+ */
+public class CommandStatus {
+
+    private ExitValue exitStatus;
```

Comment:
Does it make sense to rename the ExitValue class as ExitStatus since a status seems to be returned